### PR TITLE
General appearance modifications

### PR DESCRIPTION
**Pull request by nevesnunes**
This adds the option for different sizes in icons and fonts (issue #3), but also contains further appearance tweaks, addressing the following points:

- Highlight was being applied behind the labels, so if for example labels were to use a solid background color, the highlight would be invisible. Now this styling is applied to box layout and each of it's boxes. This gives better control in the creation of custom themes.
- There was a 1px gap between entry box and labels, which was removed by styling the box layout;
- The entry box is larger, by sharing the same font size with labels; 
- Some colors were adjusted (such as highlight) to be closer to other GNOME Shell elements.

I understand some of these are a matter of taste, so I can change some back if you prefer.

I've attached an image to illustrate these changes:
switcher

### CODE REVIEW

**Comment by daniellandau** [comment]
The image looks good. I'll take a look at the code in the evening.

**Comment by daniellandau** [comment]
This is great, thanks already! I'll add a couple of small gripes I have as line comments.

File: extension.js [hunk 1]
```diff
@@ -158,13 +163,12 @@ function _showUI() {
   let monitor = Main.layoutManager.primaryMonitor;
   container.set_width(monitor.width);
   container.set_height(monitor.height);
-  container.set_position(monitor.x, monitor.y);
+  container.set_position(monitor.x, 200);
```
Comment (daniellandau) [inline extension.js]: Here the purpose of setting the container to take up the whole screen is that clicking anywhere on the container closes the switcher. The point there is that I want to make it as unlikely as possible that the user feels stuck with the switcher, as there is no visual indication of how to get rid of it (pressing Enter and Esc are two ways besides clicking anywhere, but it's best to play it safe with something that prevents any other use of the computer).

With this change clicking at the top of the screen doesn't do anything.

File: prefs.js [hunk 2]
```diff
@@ -17,17 +17,31 @@ function init() {
...
+  let vwidget = new Gtk.VBox({margin: 10});
+
+  addShortcut(vwidget, settings);
+  addImmediately(vwidget, settings);
+
+  let hwidget = new Gtk.HBox();
+  
+  let fontSizeWidget = new Gtk.VBox({'margin-right': 10});
+  addFontSize(fontSizeWidget, settings);
+  fontSizeWidget.show_all();
```
Comment (daniellandau) [inline prefs.js]: These extra `show_all`s are not needed, the single invocation on `vwidget` below recursively shows everything.

File: stylesheet.css [hunk 3]
```diff
@@ -1,34 +1,33 @@
...
+  border-top: 2px solid #343a3a;
 }
-.switcher-box:last-child .switcher-label:last-child {
-  border-radius: 0 0 9px 0;
+.switcher-box:last-child {
+  border-radius: 0 0 9px 9px;
 }
 
 .switcher-box-layout {
-  padding-top: 100px;
```
Comment (daniellandau) [inline stylesheet.css]: This is the counterpart to the `container.set_position(monitor.x, 200);` above.

**Comment by nevesnunes** [comment]
Good point, I didn't notice that change in the container's behaviour. Since padding is affected by the background color, I used the margin property instead. Everything seems ok on the latest commit.

**Comment by daniellandau** [comment]
Great work, thank you!

**Pull request merged**